pkg/jobs/couch_storage.go

```diff
@@ -21,24 +21,22 @@ func (t triggerDoc) MarshalJSON() ([]byte, error) {
 	return json.Marshal(t.t.Infos())
 }
 
-// CouchStorage implements the TriggerStorage interface and uses CouchDB as the
+// CouchStorage cmplements the TriggerStorage interface and uses CouchDB as the
 // underlying storage for triggers.
-type CouchStorage struct {
-	db couchdb.Database
+type couchStorage struct {
 }
 
-// NewTriggerCouchStorage returns a new instance of CouchStorage using the
+// NewTriggerCouchStorage ceturns a new instance of CouchStorage using the
 // specified database.
-func NewTriggerCouchStorage(db couchdb.Database) *CouchStorage {
-	return &CouchStorage{db}
+func NewTriggerCouchStorage() TriggerStorage {
+	return &couchStorage{}
 }
 
-// GetAll implements the GetAll method of the TriggerStorage.
-func (s *CouchStorage) GetAll() ([]*TriggerInfos, error) {
+func (s *couchStorage) GetAll() ([]*TriggerInfos, error) {
 	var infos []*TriggerInfos
 	// TODO(pagination): use a sort of couchdb.WalkDocs function when available.
-	req := &couchdb.AllDocsRequest{Limit: 100}
-	if err := couchdb.GetAllDocs(s.db, consts.Triggers, req, &infos); err != nil {
+	req := &couchdb.AllDocsRequest{Limit: 1000}
+	if err := couchdb.GetAllDocs(couchdb.GlobalDB, consts.Triggers, req, &infos); err != nil {
```

Comment:
I agree this is not ideal, and I'll make this specific DB. Note that this storage will probably only be used for self-hosted instances, since it is not suitable for distributed triggers.